pruner/README.md

```diff
@@ -0,0 +1,27 @@
+# `DelPattern`
+
+The core quamina.Matcher doesn't currently support deleting patterns.
+Some of the contemplated implementations of would probably be pretty
+difficult.  At least one approach is pretty easy: Wrap the current
+matcher to filter removed patterns from match results and periodically
+rebuild the matcher from scrach with the live patterns.  More
+specifically:
+
+1. Remember patterns that have been added
+2. Remember patterns that have been removed (implicitly)
```

Comment:
what does "implicitly" mean… I think you have to remember them so you can filter them out of Match results?

Reply:
The `MemState` is just a map of the live patterns, so there is no need to remember the identities of the ones that were deleted. But that is an implementation detail of that that one implementation of `State`, and those statements are confusing regardless. I'll reword. Maybe replace both statements with just

1. Remember the set of live patterns.

Reply:
D'oh, you just filter anything that isn't in the patterns-I-know-about list. Don't mind me.